Sources/swift-format/Subcommands/LintFormatOptions.swift

```diff
@@ -98,6 +98,15 @@ struct LintFormatOptions: ParsableArguments {
   )
   var followSymlinks: Bool = false
 
+  @Option(
```

Comment:
Sorry I didn't see hit PR go by, using `String` here is not great because the user has no idea what options are valid. Instead using a custom type and conforming it to `ExpressibleByArgument` and `CaseIterable` will allow `ArgumentParser` to provide a list of valid options in the help cli and error output.

Reply:
As mentioned in the PR description, we would need `Parser.ExperimentalFeatures` to become non-SPI for that to be feasible, first. Otherwise, we can't pass the values down through the various API layers because they're `public` methods and thus would also need to be hidden behind their own SPI.

Reply:
Ah thats unfortunate, but good to know!